python/ray/tests/test_symbol_collisions.py

```diff
@@ -0,0 +1,29 @@
+"""
+This script ensures that various libraries do not conflict with ray by
+trying to import both libraries in both orders.
+A specific example is that importing ray after pyarrow causes a Segfault.
+"""
+import subprocess
+
+TESTED_LIBRARIES = ["pyarrow"]
+
+
+def test_imports():
+    def try_imports(library1, library2):
```

Comment:
@ijrsvt slightly simpler way to do this (and I think will give a better error message)

```python
subprocess.check_output(["python", "-c", "import {}; import {}".format(library1, library2)]) 
subprocess.check_output(["python", "-c", "import {}; import {}".format(library2, library1)]) 
```

Reply:
Ahh okay. I'll switch to this!